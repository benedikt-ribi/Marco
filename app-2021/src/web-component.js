class MyElement extends HTMLElement {
  connectedCallback() {
    this.innerHTML = "<b>Title</b>";
  }
}

customElements.define("my-element", MyElement);
