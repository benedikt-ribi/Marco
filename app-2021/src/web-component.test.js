/**
 * @jest-environment jsdom
 */

import { describe, expect, test } from "@jest/globals";
import "./web-component";

describe("Application", () => {
  test("initialize", async () => {
    document.body.innerHTML = `
      <my-element></my-element>
    `;

    const [myElement] = document.getElementsByTagName("my-element");

    expect(myElement.innerHTML).toMatchSnapshot();
  });
});
