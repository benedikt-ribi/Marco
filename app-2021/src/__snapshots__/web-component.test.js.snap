// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application initialize 1`] = `""`;
